src/plone/staticresources/upgrades/__init__.py

```diff
@@ -0,0 +1,21 @@
+from plone import api
+from plone.registry import field
```

Comment:
`black` complains about this file, you can use `tox -e lint` to fix that and commit it 👍🏾